src/js/helpers.js

```diff
@@ -203,7 +203,8 @@ function assignments (node) {
     [...node.querySelector('table[align=center').querySelectorAll('tr')].slice(1, -1).forEach((e, i) => {
         const curr = e.querySelectorAll('td');
         const assignment = new Assignment(curr[2]?.innerText || "", curr[curr.length - 1]?.innerText || "", i);
-        if (e.querySelector('img[src="/images/icon_missing.gif"]')) {
+        const missingIcon = e.querySelector('img[src="/images/icon_missing.gif"]');
+        if (missingIcon && missingIcon.style.display !== 'none') {
```

Comment:
You can simply this to the following, actually:
```suggestion
        if (missingIcon?.style.display !== 'none') {
```

Reply:
I don't think this actually works here as for different classes(the school kind) it seems to be inconsistent whether that CSS is present. So in some cases, a class may not have the missing indicator and in other cases, they'll have it hidden and in other cases, they'll have it and it'll be unhidden.

Reply:
I don't think this actually works here as for some classes this will be null due to inconsistent CSS(some classes don't contain the missingicon gif and some do), and as such it should be counted as not missing if it doesn't exist.